src/main/scala/com/twitter/diffy/lifter/JsonLifter.scala

```diff
@@ -25,7 +25,7 @@ object JsonLifter {
     case JsonToken.START_ARRAY        =>
       node.elements.toSeq.map {
         element => lift(element)
-      }  sortBy { _.toString }
```

Comment:
Sorting is meant to prevent false positives caused by inconsistent orders in which the same Set can be rendered as a Json array. Since Json does not have any notion of Set we have to sort just in case the collection may have originally been a Set. The down side is that this creates a blind spot for OrderingDifferences in real lists. i.e. we have false negatives instead of false positives.